{{cookiecutter.module_name}}/org_repo.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Would strongly prefer to keep this workflow in python, and maintain cross-platform compatibility (Windows, Linux, MacOS)

Reply:
moved all of this logic into the post hook